dwitter/templates/dweet/dweet.html

```diff
@@ -144,15 +185,21 @@
       if (playing){
         requestAnimationFrame(loop);
       }
-      time = frame/60;
-      if(time * 60 | 0 == frame - 1){
+      time = frame/FPS;
```

Comment:
As mentioned earlier; for some dweets, changing this when recording a gif will make the dweet behave differently. Which is not ideal. However, I don't think too many dweets are affected.

I say we do one of two things:
1. Say that this is fine for now, and if there are issues with some dweets we will look into it later
2. If you change the gif FPS to 30 or 15 you could do `if(recording && frame % 2 == 0)` to capture every other frame and still run the dweet exactly as expected.  (15 might be enough?)

What do you think?